src/main/java/biz/paluch/logging/gelf/GelfMessageAssembler.java

```diff
@@ -86,43 +86,43 @@ public void initialize(PropertyProvider propertyProvider) {
             host = propertyProvider.getProperty(PropertyProvider.PROPERTY_GRAYLOG_HOST);
         }
 
-        String port = propertyProvider.getProperty(PropertyProvider.PROPERTY_PORT);
-        if (port == null) {
-            port = propertyProvider.getProperty(PropertyProvider.PROPERTY_GRAYLOG_PORT);
+        String myPort = propertyProvider.getProperty(PropertyProvider.PROPERTY_PORT);
```

Comment:
Introducing `my…` can help to solve field shadowing. At the same time, it contributes to reading complexity as that's not the style of our code.

Reply:
Yes, this is for sure a question of taste IMHO. Shadowing doesn't hurt if you're a bit experienced and you have an IDE. 
I can live with shadowing and let SonarLint detect this.
Some projects insist on having all static code analysis metrics green :-).